Modules/audioop.c

```diff
@@ -300,13 +302,13 @@ static const int stepsizeTable[89] = {
 #ifdef WORDS_BIGENDIAN
 #define GETINT24(cp, i)  (                              \
         ((unsigned char *)(cp) + (i))[2] +              \
-        (((unsigned char *)(cp) + (i))[1] << 8) +       \
-        (((signed char *)(cp) + (i))[0] << 16) )
+        (((unsigned char *)(cp) + (i))[1] * (1 << 8)) + \
```

Comment:
Just to confirm, there's no change needed on this line other than for consistency, right? The next line is the one that can invoke undefined behaviour.

Reply:
Let me double check this one.

I think I mostly changed them for consistency, and because I didn't trust myself to count the parens right to convince myself that no UB is possible.

Reply:
Changing for consistency is fine here - it would be weird to be spelling the 8-bit shift and the 16-bit shift substantially differently.

Reply:
OK, I left this changed.